io/net/infiniband/ib_bw_perf.py

```diff
@@ -40,10 +40,17 @@ def setUp(self):
         check the availability of perftest package installed
         perftest package should be installed
         '''
-        sm = SoftwareManager()
-        depends = ["openssh-clients", "perftest"]
-        for pkg in depends:
-            if not sm.check_installed(pkg) and not sm.install(pkg):
+        smm = SoftwareManager()
+        detected_distro = distro.detect()
+        pkgs = ["perftest"]
+        if detected_distro.name == "Ubuntu":
+            pkgs.append('openssh-client')
+        elif detected_distro.name == "SuSE":
+            pkgs.append('openssh')
+        else:
+            pkgs.append('openssh-clients')
```

Comment:
is it not better to check specifically with Red-Hat/Rhel, rather than executing in else.